### PR TITLE
[ovs offload] script name change ovs_offload_lnv.py to ovs_offload_lnw.py

### PR DESCRIPTION
**Pull request by naveenashok211**
Script names updated to 
ovs_offload_lnv.py -> ovs_offload_lnw.py
ovs_offload_lnv2.py -> ovs_offload_lnw_v2.py
ovs_offload_lnv3.py -> ovs_offload_lnw_v3.py

The README is updated with the name change

### CODE REVIEW

File: ovs_offload/README.md [hunk 1]
```diff
@@ -153,12 +154,12 @@ source venv/bin/activate
 
 ## Test Script:
 
-### ovs_offload_lnv.py : (P4:fxp-net_linux-networking.p4, IPU SDK Release >= 1.7.0)
+### ovs_offload_lnw.py : (P4:fxp-net_linux-networking.p4, IPU SDK Release >= 1.7.0)
 
-1. This is a python script : **ovs_offload/ovs_offload_lnv.py** that can be used with **P4: fxp-net_linux-networking.p4** for release 1.7.0
+1. This is a python script : **ovs_offload/ovs_offload_lnw.py** that can be used with **P4: fxp-net_linux-networking.p4** for release 1.7.0
```
Comment (5abeel) [inline ovs_offload/README.md:159]:
```suggestion
1. This is a python script : **ovs_offload/ovs_offload_lnw.py** that can be used with **P4: fxp-net_linux-networking.p4** for release 1.7.0 and later
```

**Review by 5abeel — APPROVED** [review]
One minor commment. Rest looks good

**Comment by naveenashok211** [comment]
Updated the README with suggested change .
-1. This is a python script : **ovs_offload/ovs_offload_lnw.py** that can be used with **P4: fxp-net_linux-networking.p4** for release 1.7.0
+1. This is a python script : **ovs_offload/ovs_offload_lnw.py** that can be used with **P4: fxp-net_linux-networking.p4** for release 1.7.0 and later

**Review by 5abeel — APPROVED** [review]
LGTM

File: ovs_offload/README.md [hunk 2]
```diff
@@ -346,51 +367,57 @@ E20240414 00:03:48.469659 293611 es2k_hal.cc:276] [secure mode] Stratum external
...
 
-[root@ipu-acc lnv_ovs_scripts]# ./4_acc_p4rt_dump.sh
+```bash
+[root@ipu-acc ovs_offload_lnw_scripts]# pwd
+/opt/p4/p4sde/p4_test/ovs_offload_lnw_scripts
+
+[root@ipu-acc ovs_offload_lnw_scripts]# ./4_acc_p4rt_dump.sh
 ```
 
 ### 4. Setup ACC environment for OVS
```
Comment (ffoulkes) [inline ovs_offload/README.md:387]:
```suggestion
### 4. Set up ACC environment for OVS
```

File: ovs_offload/README.md [hunk 3]
```diff
@@ -346,51 +367,57 @@ E20240414 00:03:48.469659 293611 es2k_hal.cc:276] [secure mode] Stratum external
...
 
 - ACC Terminal 2 : Setup the OVS Environment.
```
Comment (ffoulkes) [inline ovs_offload/README.md:389]:
```suggestion
- ACC Terminal 2 : Set up the OVS Environment.
```

File: ovs_offload/README.md [hunk 4]
```diff
@@ -346,51 +367,57 @@ E20240414 00:03:48.469659 293611 es2k_hal.cc:276] [secure mode] Stratum external
...
 
-[root@ipu-acc lnv_ovs_scripts]# ./5_acc_setup_ovs.sh
+```bash
+[root@ipu-acc ovs_offload_lnw_scripts]# pwd
+/opt/p4/p4sde/p4_test/ovs_offload_lnw_scripts
+
+[root@ipu-acc ovs_offload_lnw_scripts]# ./5_acc_setup_ovs.sh
 ```
 
 ### 5. Setup OVS Bridge Configuration
```
Comment (ffoulkes) [inline ovs_offload/README.md:398]:
```suggestion
### 5. Set up OVS Bridge Configuration
```

File: ovs_offload/README.md [hunk 5]
```diff
@@ -346,51 +367,57 @@ E20240414 00:03:48.469659 293611 es2k_hal.cc:276] [secure mode] Stratum external
...
 
 - ACC Terminal 2 : Setup the OVS Bridge Config
```
Comment (ffoulkes) [inline ovs_offload/README.md:400]:
```suggestion
- ACC Terminal 2 : Set up the OVS Bridge Config
```

File: ovs_offload/README.md [hunk 6]
```diff
@@ -346,51 +367,57 @@ E20240414 00:03:48.469659 293611 es2k_hal.cc:276] [secure mode] Stratum external
...
-[root@ipu-acc lnv_ovs_scripts]# ./6_acc_ovs_bridge.sh
+```bash
+[root@ipu-acc ovs_offload_lnw_scripts]# pwd
+/opt/p4/p4sde/p4_test/ovs_offload_lnw_scripts
+
+[root@ipu-acc ovs_offload_lnw_scripts]# ./6_acc_ovs_bridge.sh
 ```
 
-### 6. Setup VMs on the IPU Host and configure link partner interface:
+### 6. Setup VMs on the IPU Host and configure link partner interface
```
Comment (ffoulkes) [inline ovs_offload/README.md:409]:
```suggestion
### 6. Set up VMs on the IPU Host and configure link partner interface
```

File: ovs_offload/README.md [hunk 7]
```diff
@@ -412,20 +439,23 @@ ip netns exec VM4 ping 20.0.0.30
 ### 7. Optional: Setup OVS Bridge for VXLAN
```
Comment (ffoulkes) [inline ovs_offload/README.md:439]:
```suggestion
### 7. Optional: Set up OVS Bridge for VXLAN
```

File: ovs_offload/README.md [hunk 8]
```diff
@@ -503,13 +533,13 @@ rtt min/avg/max/mdev = 0.040/0.048/0.058/0.007 ms
 
 ```
 
-
-### OVS Offload VXLAN on 2 IPU Peers Connected back to back:
+### OVS Offload VXLAN on 2 IPU Peers Connected back to back
```
Comment (ffoulkes) [inline ovs_offload/README.md:536]:
```suggestion
### OVS Offload VXLAN on Two IPU Peers Connected back-to-back
```

File: ovs_offload/ovs_offload_lnw_v2.py [hunk 9]
```diff
@@ -563,7 +561,7 @@ def build_p4rt_config(vf_list=[], acc_pr_list=[],vm_ip_list=[], host_idpf_intf='
 
 def build_args():
     # Create the top-level parser
-    parser = argparse.ArgumentParser(description='Run Linux networking with OVS Offload')
+    parser = argparse.ArgumentParser(description='Run Linux networking V2 with OVS Offload')
```
Comment (ffoulkes) [inline ovs_offload/ovs_offload_lnw_v2.py:564]: We've dropped support for LNW V2.

**Review by ffoulkes — COMMENTED** [review]
See suggested changes

I think I've marked all the incorrect uses of "setup."

File: ovs_offload/ovs_offload_lnw_v2.py (hunk 9 above)
Comment (naveenashok211) [reply ovs_offload/ovs_offload_lnw_v2.py:564]: We have 3 versions of the script to support the OVS offload recipe on all releases from 1.4 to 1.7 as we had customers using the older versions like 1.4 for the POC hence I have retained the 3 versions , but we can remove it in future once customers move to a release 1.7 and above

ovs_offload_lnw.py : (P4:fxp-net_linux-networking.p4, IPU SDK Release >= 1.7.0)
ovs_offload_lnw_v3.py : (P4:fxp-net_linux-networking_v3.p4, IPU SDK Release 1.6.0, 1.6.1)
ovs_offload_lnw_v2.py: (P4:fxp-net_linux-networking_v2.p4, IPU SDK Release 1.4.0)

File: ovs_offload/README.md [hunk 10]
```diff
@@ -1,27 +1,29 @@
...
-configure OVS and create OVS bridges and configure VMs on the IDPF interfaces on the Host.
+- The ovs_offload_lnw.py script can be used on an host server connected with Intel® Infrastructure Processing Unit via PCIe.
+- It creates the configuration to run the Linux Networking Recipe with OVS Offload on the ACC.
+- It can be used to start infrap4d on the ACC and use the script generated p4rt-ctl rules to configure ACC Port representors for the Host IDPF interfaces and IPU Physical Ports.
+- It can setup OVS bridges on the ACC using the port representors and configure VMs on the IDPF interfaces on the Host.
 
-## Test Environment Setup:
+## Test Environment Setup
 
 - Before running script check if ssh as a root user is enabled and root user password is setup for ssh.
```
[the remark this answers is not in the document]
Comment (naveenashok211) [reply ovs_offload/README.md]: Addressed this in the latest commit .

File: ovs_offload/ovs_offload_lnw_v2.py (hunk 9 above)
Comment (ffoulkes) [reply ovs_offload/ovs_offload_lnw_v2.py:548]: Ah. Got it.

**Pull request merged**